MediaGalleryUi/view/adminhtml/web/js/grid/columns/image.js

```diff
@@ -17,7 +17,7 @@ define([
             deleteSelectedBtnSelector: '#delete_selected',
             targetElementId: null,
             selected: null,
-            mediaGalleryDirectoryComponent: 'media_gallery_listing.media_gallery_listing.media_gallery_directories',
+            mediaGalleryDirectoryComponent: '${ $.ns }.${ $.ns }.media_gallery_directories',
```

Comment:
It would be better to move name space to xml configuration

Reply:
https://github.com/magento/adobe-stock-integration/blob/36b8d15c61f168969aa39a1e8539cd20a24cf902/MediaGalleryUi/view/adminhtml/ui_component/media_gallery_listing.xml#L178